routes/snippets.js

```diff
@@ -1,4 +1,42 @@
 const express = require("express");
 const router = express.Router();
 
+const mongoose = require("mongoose");
+const createError = require("http-errors");
+
+const Snippet = require("../models/Snippet");
+
+const {
+  OK,
+  INVALID_ID,
+  INVALID_REQUEST,
+  INTERNAL_SERVER_ERROR,
+} = require("../constants/messages");
+
+router.get("/:id", async (req, res, next) => {
+  const SnippetId = req.params.id;
```

Comment:
소문자로 시작해주세요!

Reply:
다음과 같이 비구조화하여 사용할 수 있습니다.
```suggestion
  const { id } = req.params;
  // or
  const { SnippetId: id } = req.params;
```
한나님은 비구조화하여 작성하고 계셔서, 이 부분도 어떤 식으로 스타일을 맞출지 상의하면 좋을 것 같습니다.

Reply:
비구조화로 작성하겠습니다~!